app/src/main/java/com/example/muvi/data/model/MovieItem.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.muvi.data.model
+
+import com.google.gson.annotations.SerializedName
+
+data class MovieItem(
+    @SerializedName("id")
+    val id: Int,
+    @SerializedName("title")
+    var title: String,
+    @SerializedName("vote_average")
+    var rate: Float,
+    @SerializedName("overview")
+    var description: String,
+    @SerializedName("poster_path")
+    var poster: String?,
+    @SerializedName("backdrop_path")
+    var background: String,
+    @SerializedName("release_date")
+    var releaseDate: String,
+    @SerializedName("runtime")
+    var runtime: Int,
+    @SerializedName("genre_ids")
+    var genreId: List<Int>?,
+    @SerializedName("genres")
+    var genre: List<GenreItem>?,
+    @SerializedName("production_companies")
+    var company: MutableList<CompanyItem>?
```

Comment:
```suggestion
    var company: List<CompanyItem>?
```
sao những thuộc tính này lại cần nullable?

Reply:
vâng, e nghĩ cũng k cần nullable